u2f-chrome-extension/jasmine/spec/SoftTokenHelperSpec.js

```diff
@@ -0,0 +1,365 @@
+describe("Spec testing", function() {
+  var pubKey, secKey;
+  var appIdHash, keyHandle, challengeHash, version, timeoutSeconds;
+  var enrollChallenges, signData;
+  var softTokenProfile, softTokenHelper;
+  var softTokenSignHandler, softTokenEnrollHandler;
+  var enrollHelperRequest, signHelperRequest;
+  var expectedEnrollResponseAlreadyMatchingKeyHandle;
+  var originalKeyHandle, newKeyHandle, newPubKey, newSecKey;
+  var exampleValidTransferAccessMessage, exampleChainedTransferAccessMessage;
+  var exampleRegistration, exampleTransferAccess, exampleTransferAccessChain;
+  var expectedSoftTokenSignResponseNoMatchingKey;
+  var expectedSoftTokenSignResponseMatchingKey;
+
+  beforeEach(function() {
+    pubKey =
+      "041155B2F73ABECCD83986C65AAC0D0BA709762AF2194DA800851026BC828E39C069FF8B\
+       40F765CD6E36FF164CD31CB96EA7D4326A3C635B3BCDB6AA45E2FEF5CE";
+    secKey = "3C0D26EFA654915BAF619FD94E37084F4024CA3ED52438A2837D9ACB510051C0";
```

Comment:
I decided for readability against realism here, as the code doesn't seem to be dependent on matching generated key handles. The key handles are simply defined as "keyHandle" "newKeyHandle" and "originalKeyHandle".

This is not consistent with pubkey/secKey since I figured at some point they will be used for real signing. As such, those keys are simply copied from the console during debugging. They are generated by the software token.